src/groups/wrappers/z_bmq/z_bmqa/z_bmqa_sessionevent.t.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2014-2023 Bloomberg Finance L.P.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// bmqa_sessionevent.t.cpp                                            -*-C++-*-
+#include <z_bmqa_sessionevent.h>
+
+// TEST DRIVER
+#include <mwctst_testhelper.h>
+
+// CONVENIENCE
+using namespace BloombergLP;
+using namespace bsl;
+
+// ============================================================================
+//                                    TESTS
+// ----------------------------------------------------------------------------
+
+static void test1_breathingTest()
+{
+    mwctst::TestHelper::printTestName("BREATHING TEST");
+
+    z_bmqa_SessionEvent * obj;
+    z_bmqa_SessionEvent__create(&obj);
```

Comment:
Since you tested creation, you might as well test `z_bmqa_SessionEvent__delete`.